charactersheet/charactersheet/viewmodels/character/ability_scores.js

```diff
@@ -61,6 +58,10 @@ function AbilityScoresViewModel() {
 
     self.unload = function() {
         self.abilityScores().save();
+
+        Notifications.global.save.remove(function() {
```

Comment:
Unsubscribing like this won't work. This was a cheat (hack) that we could do because `init` was only called once.

You have to pass it the same fn you used at subscription-time. You'll have to break out this function into a proper method like below:

```js
self.load = function() {
  Notifications.global.save.add(self.save);
}
self.unload = function() {
  Notifications.global.save.remove(self.save);
}
self.save = function() {
  self.abilityScores().save();
}
```